task_processing/runners/runner.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import abc
+
+import six
+
+
+@six.add_metaclass(abc.ABCMeta)
+class Runner(object):
```

Comment:
originally runner was called concurrency policy, they exist to give users choice of concurrency style they want to interact in. task executor is probably poorly named - it just wires together framework and the driver, and has less platform-specific interface (takes dicts instead of mesos_pb objects, etc).

on folder structure - i agree that keeping all platform-specific stuff in plugin folders looks better.

can you elaborate on multi_mesos_executor.py?